docs/content/blog/ignoring-errors.md

```diff
@@ -0,0 +1,136 @@
+---
+title: "Ignoring Mixin Errors and Idempotent Actions"
+description: "Porter now supports ignoring the errors from a mixin. The az mixin takes advantage of this new feature to manage resource groups."
+date: "2022-01-07"
+authorname: "Carolyn Van Slyck"
+author: "@carolynvs"
+authorlink: "https://twitter.com/carolynvs"
+authorimage: "https://github.com/carolynvs.png"
+tags: ["mixins"]
+summary: |
+    Porter now supports ignoring the errors from a mixin. The az mixin takes advantage of this new feature to manage resource groups.
+---
+
+When is an error not really an error?
+Questions like this come up regularly when automating deployments.
```

Comment:
is this the question? (The when is an error not really an error) and more something like .. Why is my automated deployment erroring out when the resources exist? or something else. It's almost like when is an error not really an error is presupposing folks understanding what's up with the feature that based on me not being as steeped in what's going on with Porter really confused me at first by the title/description even.

Reply:
I'll rework it so that it's less clever and more clear. 👍